security/subsystem/src/main/java/org/jboss/as/security/SecurityExtension.java

```diff
@@ -226,6 +246,21 @@ private void registerTransformers_1_2_0(SubsystemRegistration subsystemRegistrat
         ResourceTransformationDescriptionBuilder builder = TransformationDescriptionBuilder.Factory.createSubsystemInstance();
 
         ResourceTransformationDescriptionBuilder securityDomain = builder.addChildResource(SECURITY_DOMAIN_PATH);
+        securityDomain.getAttributeBuilder().setDiscard(new DiscardAttributeChecker.DiscardAttributeValueChecker(false, true, new ModelNode(false)),
+                SecurityDomainResourceDefinition.EXPORT_ELYTRON_REALM)
+                .addRejectCheck(new RejectAttributeChecker.DefaultRejectAttributeChecker() {
+
+                    @Override
+                    public String getRejectionLogMessage(final Map<String, ModelNode> attributes) {
+                        return SecurityLogger.ROOT_LOGGER.rejectedAttributeMustUndefinedOrFalse();
+                    }
+
+                    @Override
+                    protected boolean rejectAttribute(final PathAddress address, final String attributeName, final ModelNode attributeValue, final TransformationContext context) {
+                        return attributeValue.asBoolean();
+                    }
+                }, SecurityDomainResourceDefinition.EXPORT_ELYTRON_REALM)
```

Comment:
You have the discard above which will discard 'false', so you can reject always. This looks like the scenario discussed here:

https://docs.jboss.org/author/display/WFLY10/Domain+Mode+Subsystem+Transformers#DomainModeSubsystemTransformers-Defaultvalueoftheattributeisthesameaslegacyimpliedbehaviour